src/provider/__tests__/RUIProvider.test.jsx

```diff
@@ -36,4 +37,77 @@ describe('rendering', () => {
 
     assert(dom.container.firstChild);
   });
+
+  it('renders with nested providers', () => {
+    const dom = render((
+      <RUIProvider
+        globalProps={{
+          Grid: {
+            alignContent: {
+              sm: 'column',
+              xs: 'row dense',
+            },
+            autoFlow: {
+              sm: 'column',
+              xs: 'row dense',
+            },
+            justifyItems: 'center',
+            tag: 'main',
+          },
+        }}
+      >
+        <RUIProvider
+          globalProps={{
+            Grid: {
+              alignContent: undefined,
+              autoFlow: {
+                lg: 'column',
+                sm: undefined,
+                xs: 'row dense',
+              },
+              justifyContent: undefined,
+              justifyItems: undefined,
+              tag: 'section',
+            },
+          }}
+        >
+          <RUIProvider
+            globalProps={{
+              Grid: {
+                autoFlow: {
+                  md: 'column',
+                },
+                justifyContent: 'center',
+              },
+            }}
+          >
+            <Grid>
+              <div>
+                Content text
+              </div>
+            </Grid>
+          </RUIProvider>
+        </RUIProvider>
+      </RUIProvider>
+    ));
+
+    // Assert alignContent
+    expect(dom.container.firstChild.style.cssText.includes('--rui-local-align-content')).toBeFalsy();
```

Comment:
Is truthy/falsy sufficient? Can we be sure that e.g. `column` is correctly overwritten to `row dense`?

Reply:
Is is sufficient to compare it to false as `alignContent` is reset via `undefined`.